packages/v4/src/content/design-guidelines/components/progress/progress.md

```diff
@@ -3,102 +3,187 @@ id: Progress
 
-![progress-bar](./img/progress-bar-in-table.png)
+### Error state or failure
 
-If the progress bar records the progress of multiple line items, keep the progress bar in-line with line item it pertains to.
+A red progress bar represents a process that has failed. Accompany a failed progress bar with a [red danger icon](/guidelines/icons/#all-icons) to demonstrate that an error occurred in the process.
 
-**How to use a progress bar in a dashboard**
+<img src="./img/progress-bar-error-state.png" alt="An example of a progress bar in an error or failed state, with a red track, optional danger icon, and a title that indicates its status: 'Could not validate account credentials.'" />
 
-![progress-bar](./img/progress-bar-in-dashboard.png)
+### Complete or success
 
-On the dashboard view, use a progress bar to track the trends within the card components on the page.
+A green progress bar represents the successful completion of a process. Accompany a complete progress bar with a [green check-circle icon](/guidelines/icons/#all-icons) to demonstrate that the process has finished with no errors.
 
-**How to use a progress bar when a file is downloading**
+<img src="./img/progress-bar-complete.png" alt="An example of a progress bar in a complete or successful state, with a green track, optional check-circle icon, and a title that indicates its status: 'Validated account credentials.'" />
 
-![progress-bar](./img/progress-bar-file-downloading.png)
+## Spacing and alignment
 
-When a file is downloading, use the toast notification component and a progress bar to demonstrate the progress achieved.
+Right-align progress value text when it’s placed above the progress track, in line with the title. 
 
-**How to use a progress bar in a wizard**
+<img src="./img/progress-bar-right-aligned-progress-value.png" alt="A visual example of a progress bar with right-aligned progress value text above the progress track." />
 
-![progress-bar](./img/progress-bar-wizard.png)
+Left-align progress value text when it’s placed to the right of the track.
 
-Use a progress bar in a wizard to demonstrate to the user where they are in the flow.
+<img src="./img/progress-bar-left-aligned-progress-value.png" alt="A visual example of a progress bar with left-aligned progress value text in line with the progress track." />
 
-## Styling
-Styling for the progress bar should follow accessibility guidelines. The styling demonstrated below should guide design decisions.
+Always set an “outside fixed width measure” for determinate progress bars. This fixed spacing should solve alignment use cases in which stacked progress bar values vary in length. 
 
-A blue progress bar represents a process in progress.
+## Placement
 
-![progressbar](./img/progress-bar-blue.png)
+<img src="./img/progress-bar-do.png" alt="A visual example of how to successfully place a progress bar in your designs, aligned with corresponding content." />
```

Comment:
Could we maybe combine these two images into one? And make the grey padding around them smaller so that more fits in a smaller space if that makes sense. Especially if you're worried that the image will be too small if they're combined?

My worry is that the "Do" and "Don't" headers will be missed